src/lib/config.ts

```diff
@@ -115,13 +115,6 @@ class Config {
    * @param {IConfig} - config object to store as config file
    */
   private async write(config: IConfig): Promise<void> {
```

Comment:
where do we create a directory instead?

Reply:
https://github.com/subspace/subspace-desktop/pull/342/files/4271bed595cb7af0703640e21551d7225adb2faf#diff-0b4f5977a52cef50ed20e7cdf9761795df9ee1cb3b5e5ea45edeb9394e8a47ffR65

here, trail of bits reported we are creating the config file in 2 different locations, so I kept the one in `init()`